crypto/cms/cms_local.h

```diff
@@ -47,8 +47,8 @@ typedef struct CMS_ReceiptsFrom_st CMS_ReceiptsFrom;
 typedef struct CMS_CTX_st CMS_CTX;
 
 struct CMS_CTX_st {
-    OSSL_LIB_CTX *libctx;
-    char *propq;
+    OSSL_LIB_CTX *cms_libctx;
+    char *cms_propq;
```

Comment:
Why the name change?  They already sit inside the CMS context structure, so that extra prefix seems rather redundant...

Reply:
I did this to make sure I did not leave any direct access to the variables.. Changed the names back now that this has been verified.